.github/workflows/apex-nightly.yml

```diff
@@ -16,8 +16,8 @@ jobs:
       e2e_polybft_test: true
       e2e_apex_test: true
       apex_bridge_ref: main
-      e2e_apex_big_test: ${{ vars.RUN_E2E_BIG_TESTS }}
```

Comment:
this file is for nightly CI. it doesnt affect PR CI, so the CI for this PR ran without big tests and redundant tests were skipped